tsconfig.json

```diff
@@ -2,17 +2,17 @@
   "compilerOptions": {
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "target": "ES2019" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
```

Comment:
Why bump this up to ES2019?

Reply:
I'll switch it back and test to remember. There was some syntax I was using that barked and said I needed 2019.

Reply:
It's `flat` I was trying to use

Reply:
👍

Reply:
Probably. When I get back later, I'll try to use `reduce` instead.